webapp/src/components/login.module.scss

```diff
@@ -60,11 +60,9 @@
 
           .text,
           .text2 {
-            font-family: "PingFang SC";
```

Comment:
i think we can just get rid of `.text2{}` and `.text1{}` because its body is empty